www/index.rst

```diff
@@ -223,24 +226,12 @@ NumPy 1.11.2 released :news-date:`2016-10-03`
 SciPy 0.18.0 released :news-date:`2016-07-25`
     See :doc:`/scipylib/download`.
 
-NumPy 1.11.1 released :news-date:`2016-06-26`
-    See :doc:`/scipylib/download`.
-
```

Comment:
Instead of deleting all these announcements, can you move them to ``past-news.rst``?